src/react/components/_academics/StudentProfile/Degrees.js

```diff
@@ -9,9 +9,11 @@ import { ICON_GRADUATION, ICON_CERTIFICATE } from '../../Icon/IconTypes';
 import './Degrees.scss';
 
 const college = degree => degree.plans[0].college;
-const description = degree => degree.academicDegree.type.description;
+// academic degree code 28 = Law LLM, 69 = Law JSD
+const description = degree => (degree.academicDegree.type.code == "28") ? "" : degree.academicDegree.type.description;
```

Comment:
@frankiepun - Couldn't you define these as [constants](https://www.w3schools.com/js/js_const.asp) to avoid needing to use the comment to note what the codes represent?

```javascript
const LAW_LLM = "28";
const LAW_JSD = "69";
const description = degree => (degree.academicDegree.type.code == LAW_LLM) ? "" : degree.academicDegree.type.description;
const inWord = degree => (degree.academicDegree.type.code == LAW_LLM || degree.academicDegree.type.code == LAW_JSD) ? "" : "in";
```

Reply:
Great suggestion! I like it. I will redo and check in again. Thanks!

Reply:
@redconfetti, I changed it based on your suggestion. Please review again. Thanks for the great feedback!